src/cli/serve.js

```diff
@@ -29,6 +30,7 @@ const serveCommand: Command = async (args, std) => {
   }
 
   const server = express();
+  server.use(cors());
```

Comment:
This introduces a security vulnerability.

Your blanket `cors()` applies to all routes, including the `POST` route
for `/data/ledger.json`. This means that if you have an instance admin
server running, and you visit *any website*, then that website can
overwrite your ledger with arbitrary data.

Repro: run the `cd sharness/__snapshots__/test-instance` instance and
open both <http://localhost:6006> and <https://example.com>. Execute the
following JavaScript statement in your console:

```javascript
await fetch("http://localhost:6006/data/ledger.json", {method: "POST", body: "test"})
```

Before this patch, this works from localhost (same-origin) but fails
from example.com. By “fails”, I mean both that the promise rejects and
also that the file is not overwritten. After this patch, example.com is
able to overwrite the data.

The same-origin policy is a security mechanism. Please be careful and
thoughtful when working around it.